src/index.js

```diff
@@ -0,0 +1,203 @@
+    backgroundSunny: 'Current sky is sunny',
+    backgroundCloudy: 'Current sky is cloudy',
+    backgroundRaining: 'Current sky is raining',
+    backgroundSnowing: 'Current sky is snowing',
+};
+const state = {
+    city: 'Baltimore',
+    temp: 70,
+    tempColour: 'black',
+    foreground: images.foregroundSpring,
+    backgroundImage: images.backgroundSunny,
+    foregroundAccessible: accessibleText.foregroundSpring,
+    skyAccessible: accessibleText.backgroundSunny,
+};
+
+const increaseTemp = () => {
+    state.temp += 1;
+    updateTemp();
+};
+
+const decreaseTemp = () => {
+    state.temp -= 1;
+    updateTemp();
+};
+
+const updateTemp = () => {
+    const temp = document.getElementById('temp');
+    const foreground = document.getElementById('weatherGarden');
+    const foregroundAccessible = document.getElementById(
+        'weatherGardenAccessibleForeground'
+    );
+    temp.textContent = `${state.temp}°F`;
+    if (state.temp > 80) {
+        state.tempColour = '#FF9AA2';
+        state.foreground = images.foregroundHot;
+        state.foregroundAccessible = accessibleText.foregroundHot;
+    } else if (state.temp > 60) {
+        state.tempColour = '#FFDAC1';
+        state.foreground = images.foregroundSpring;
+        state.foregroundAccessible = accessibleText.foregroundSpring;
+    } else if (state.temp > 50) {
+        state.tempColour = '#E2F0CB';
+        state.foreground = images.foregroundFall;
+        state.foregroundAccessible = accessibleText.foregroundFall;
+    } else if (state.temp > 40) {
+        state.tempColour = '#C7CEEA';
+        state.foreground = images.foregroundWinter;
+        state.foregroundAccessible = accessibleText.foregroundWinter;
+    }
+    temp.style.color = state.tempColour;
+    foreground.src = state.foreground;
+    foregroundAccessible.textContent = state.foregroundAccessible;
+};
+
+const changeSky = () => {
+    const sky = document.getElementById('sky-select');
+    const selection = sky.options[sky.selectedIndex].value;
+    const skyBackground = document.getElementById('sky');
+    const skyAccessible = document.getElementById('weatherGardenAccessibleSky');
+    if (selection === 'sunny') {
+        state.skyAccessible = accessibleText.backgroundSunny;
+        state.backgroundImage = images.backgroundSunny;
+    } else if (selection === 'cloudy') {
+        state.skyAccessible = accessibleText.backgroundCloudy;
+        state.backgroundImage = images.backgroundCloudy;
+    } else if (selection === 'raining') {
+        state.skyAccessible = accessibleText.backgroundRaining;
+        state.backgroundImage = images.backgroundRaining;
+    } else if (selection === 'snowing') {
+        state.skyAccessible = accessibleText.backgroundSnowing;
+        state.backgroundImage = images.backgroundSnowing;
+    }
+    skyAccessible.textContent = state.skyAccessible;
+    skyBackground.style.backgroundImage = `url('${state.backgroundImage}')`;
```

Comment:
We might consider restructuring our data some to make access easier in functions like this. If we had a structure that was an object that held objects like so:
```javascript
const skySettings = {
    sunny: {
        accessibleText: 'Current sky is sunny',
        image: './assets/bg_sunny.png',
    },
    ... // (And so on for the other sky options)
};
```
Then here we could remove the if/else tree and do something like:
```javascript
skyAccessible.textContent = skySettings[selection].accessibleText;
skyBackground.style.backgroundImage = `url(skySettings[selection].image)`;
```

Reply:
Oh, that's so much easier to read, I didn't even realize you could do that!